modules/RTC/RTCBrowserType.js

```diff
@@ -164,8 +164,9 @@ const RTCBrowserType = {
      * @returns {boolean}
      */
     usesNewGumFlow() {
-        return RTCBrowserType.isChrome()
-            && RTCBrowserType.getChromeVersion() >= 61;
+        return (RTCBrowserType.isChrome()
+            && RTCBrowserType.getChromeVersion() >= 61)
+            || RTCBrowserType.isFirefox();
```

Comment:
let's double check if we can move all versions of firefox over.

Reply:
Drive by comment: do we have a policy on what versions of Firefox we support? I guess that supporting >= ESR should be ok, that is, >= 52. Older versions could be considered unsupported.

Reply:
not that i know of, but having a policy of something like `N versions behind stable` would definitely be nice (and make this sort of work much more straightforward)

Reply:
I uploaded this branch to my instance, used Firefox 50 and a quick sanity check passed, including screensharing with the extension. The version displayed in ESR 52.4.1.